src/lib/components/schedule/schedule-input-payload.svelte

```diff
@@ -0,0 +1,47 @@
+<script lang="ts">
+  import CodeBlock from '$lib/holocene/code-block.svelte';
+  import { translate } from '$lib/i18n/translate';
+  import type { Payloads } from '$lib/types';
+  import {
+    parseWithBigInt,
+    stringifyWithBigInt,
+  } from '$lib/utilities/parse-with-big-int';
+
+  import PayloadDecoder from '../event/payload-decoder.svelte';
+
+  export let input: string;
+  export let payloads: Payloads;
+
+  const handleInputChange = (event: CustomEvent<string>) => {
+    input = event.detail;
+  };
+</script>
+
+<div class="flex flex-col gap-4">
+  <label for="schedule-input">{translate('workflows.input')}</label>
+  <PayloadDecoder value={payloads} let:decodedValue key="payloads">
+    {@const singlePayload =
```

Comment:
I don't love having this much logic in the template code. I could even see it benefiting from having a unit test since there are a lot of hidden cyclomatic complexity.

Reply:
1000% percent agree, this is gross and I need to fix.